cmd/snap-update-ns/sorting_test.go

```diff
@@ -73,6 +73,25 @@ func (s *sortSuite) TestParallelInstancesAndSimple(c *C) {
 	})
 }
 
+func (s *sortSuite) TestOvernameOrder(c *C) {
+	expected := []osutil.MountEntry{
+		{Dir: "/a/b/2", Options: []string{osutil.XSnapdOriginOvername()}},
+		{Dir: "/a/b"},
+	}
+	entries := []osutil.MountEntry{
+		{Dir: "/a/b"},
+		{Dir: "/a/b/2", Options: []string{osutil.XSnapdOriginOvername()}},
+	}
```

Comment:
Would it make sense to have a 3rd entry that is unrelated to the two key entries?

Reply:
No, the point is to have a minimal set of entries such that the test cases of ith(overname) < jth, and ith > jth(overname) are executed. The test cases with more mixed entries are covered by other unit tests (TestParallelInstancesAndSimple and TestParallelInstancesAlmostSorted)

Reply:
Got it, thanks!